addons/mail/tests/__init__.py

```diff
@@ -10,3 +10,4 @@
 from . import test_res_partner
 from . import test_res_users_settings
 from . import test_update_notification
+from . import test_parse_inline_template
```

Comment:
Basic tests of parse inline template should be done in base, where the tool is defined.